blocks/cards/cards.js

```diff
@@ -5,14 +5,32 @@ export default function decorate(block) {
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
     const li = document.createElement('li');
+    // li.classList.add('ethos-2-theme')
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
-      if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      if (div.children.length === 1 && div.querySelector('img')) {
+        div.classList.add('card-image');
+        div.getElementsByTagName('img')[0].classList.add('icon-svg-ethos', 'lazyloaded');
```

Comment:
These classes don't seem to have any effect on the styling. Do we need to set them?

Reply:
removed them